src/EventStore.Core/Services/Transport/Grpc/Streams.Append.cs

```diff
@@ -103,21 +103,21 @@ partial class Streams {
 					return;
 				}
 
+				var response = new AppendResp();
 				switch (completed.Result) {
 					case OperationResult.Success:
-						var response = new AppendResp();
-
+						response.Success = new AppendResp.Types.Success();
 						if (completed.LastEventNumber == -1) {
-							response.NoStream = new Empty();
+							response.Success.NoStream = new Empty();
```

Comment:
This is a bit weird (but shouldn't be addressed in this PR). If we successfully appended to an empty stream we should still be able to return CurrentRevision for the next write? @thefringeninja do you know why we don't?

Reply:
I do not know